src/lib/logical_query_plan/drop_view_node.cpp

```diff
@@ -14,9 +14,9 @@ namespace opossum {
 DropViewNode::DropViewNode(const std::string& view_name, const bool if_exists)
     : BaseNonQueryNode(LQPNodeType::DropView), view_name(view_name), if_exists(if_exists) {}
 
-std::string DropViewNode::description() const { return "[Drop] View: '"s + view_name + "'"; }
+std::string DropViewNode::description(const DescriptionMode mode) const { return "[Drop] View: '"s + view_name + "'"; }
```

Comment:
Even though it's not your code, mixing this `"bla"s` with the `std::string("bla")` triggers my neuroses.

Reply:
Doesn't get mixed in this file. `std::string_literals` is used in 23 files and I don't really feel like touching all of them.